tests/test_2_neurolucida.py

```diff
@@ -510,3 +510,29 @@ def test_markers():
                                      [-268.17,  -130.62,   -24.75],
                                      [-266.79,  -131.77,   -26.13]],
                                     dtype=np.float32))
+
+
+def test_pia():
+    m = Morphology(os.path.join(_path, 'pia.asc'))
+    assert_equal(len(m.root_sections), 1)
+    assert_array_equal(m.root_sections[0].points,
+                       np.array([[-2.87, -9.24, -5.06],
+                                 [-2.76, -10.41, -5.13],
+                                 [-2.03, -12.48, -5.13],
+                                 [-1.62, -13.30, -5.56]], dtype=np.float32))
+
+    assert_equal(len(m.markers), 2)
+    pia = m.markers[0]
+    assert_equal(pia.label, 'pia')
+    assert_array_equal(pia.points,
+                       [[0, 1, 2],
+                        [0, 2, 2],
+                        [0, 3, 2],
+                        [0, 4, 2]])
+    assert_array_equal(pia.diameters, [3, 3, 3, 3])
```

Comment:
I am a bit paranoïd, but I would use distinct values instead of `[3, 3, 3, 3]`. Same for the points.

Reply:
Toujours aussi rapide le Tristan :) J'espere que le confinement se passe bien pour toi.
Done, I have changed the values.